### PR TITLE
Update using RegExr

### PR DESCRIPTION
**Pull request by AndrewJWin**
Including small grammatical fixes. There's no need to Capitalize Every Word.

### CODE REVIEW

**Comment by AndrewJWin** [comment]
Credit to @ohlookitsAugust for spotting the error first, however I had other edits that were needed to be added.

**Comment by auguwu** [comment]
<3

File: commands/games/overwatch.js [hunk 1]
```diff
@@ -17,15 +15,15 @@ class Overwatch extends Command {
 
   async run(message, [platform, location, player], level) { // eslint-disable-line no-unused-vars
     if (!player.length || !location.length || !platform.length) return message.response(undefined, `Ba....Baka! Invalid Usage, please do:\`${this.help.usage}\``);
-    if (!locationArray.includes(location)) return message.response(undefined, "Ba....Baka! Invalid Location. Valid Locations are us eu, kr, cn or global");
-    if (!platformArray.includes(platform)) return message.response(undefined, "Ba....Baka! Invalid Platform. Valid Platforms are pc, xbl or psn");
+    if (!/(us|eu|kr|cn|global)$/i.test(location)) return message.response(undefined, "Ba....Baka! Invalid Location. Valid Locations are us eu, kr, cn or global");
```
Comment (MrJacz) [inline commands/games/overwatch.js:18]: I'm seeing this as a nitpick more than an improvement...

File: commands/games/overwatch.js [hunk 2]
```diff
@@ -17,15 +15,15 @@ class Overwatch extends Command {
...
+    if (!/(xbl|psn|pc|xbox)$/i.test(platform)) return message.response(undefined, "Ba....Baka! Invalid Platform. Valid Platforms are pc, xbl or psn");
```
Comment (MrJacz) [inline commands/games/overwatch.js:19]: I'm seeing this as a nitpick more than an improvement...

File: commands/games/overwatch.js [hunk 3]
```diff
@@ -17,15 +15,15 @@ class Overwatch extends Command {
...
     if (platform === "xbox") platform = "xbl";
     player = player.replace(/#/g , "-");
     const data = await owjs.getAll(platform, location, player).catch(e => {
       console.log(e.stack);
       return null;
     });
-    if (!data) message.response("❗", "Sorry, But Something Went Wrong, please try again later.");
+    if (!data) message.response("❗", "Sorry, but something went wrong. Please try again later.");
```
Comment (MrJacz) [inline commands/games/overwatch.js:26]: Shouldn't it be `Sorry something went wrong, please try again later.`?

Comment (auguwu) [reply commands/games/overwatch.js:26]: It's a typo, that's what the PR is saying.

**Comment by YorkAARGH** [comment]
Closing this as I merged another PR that did the same thing (basically)

**Pull request closed without merging**